content/quickstart/kubernetes/configure.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Configure Kubernetes
+weight: 4
+menu:
+  quickstart:
+    parent: kubernetes
+    identifier: kubernetes-configure
+---
+
+<!-- TODO inline a streamlined version of configuring the cloud here. -->
+
+<a href="{{< relref "/reference/clouds/kubernetes/setup.md" >}}" target="_blank">Configure Kubernetes</a> so the Pulumi CLI can connect to a Kubernetes cluster.
```

Comment:
Repeating CLI comments above, say that if `kubectl` is configured to talk to your cluster, it'll just work.

Reply:
Fixed